specs/mrtcore/PrimaryLanguageOverride/PrimaryLanguageOverride.md

```diff
@@ -0,0 +1,190 @@
+# ApplicationLanguages
+
+This new type (`Microsoft.Windows.ApplicationModel.Resources.ApplicationLanguages`) is a wrapper for
+the OS `Windows.Globalization.ApplicationLanguages` type with the updated behavior for the
+`PrimaryLanguageOverride` property to be supported for un-packaged apps.
+
+# Background
+
+Changing the application language using
+[PrimaryLanguageOverride](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages.primarylanguageoverride)
+property from `Windows.Globalization.ApplicationLanguages` namespace is not supported for
+WindowsAppSDK un-packaged apps, only for packaged. To support language change for un-packaged
+applications `ApplicationLanguages` type is introduced in
+`Microsoft.Windows.ApplicationModel.Resources` MRTCore namespace.
+`Microsoft.Windows.ApplicationModel.Resources.ApplicationLanguages.PrimaryLanguageOverride` property
+is supported both for packaged and un-packaged WindowsAppSDK apps.
+
+This new type (`Microsoft.Windows.ApplicationModel.Resources.ApplicationLanguages`) is a wrapper for
+the OS type with the updated behavior for the `PrimaryLanguageOverride` to support un-packaged apps.
+`Microsoft.Windows.ApplicationModel.Resources.ApplicationLanguages` should serve as a replacement
+for the existing OS type (`Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride`).
+
+`GetLanguagesForUser(User)` method from `Windows.Globalization.ApplicationLanguages` is being
```

Comment:
The (User) part is confusing to me. Is this supposed to be (ForUser) like there are two methods one with ForUser at the end?

```suggestion
`GetLanguages(ForUser)` method from `Windows.Globalization.ApplicationLanguages` is being
```

Reply:
It's the argument of the method. The method name is GetLanguagesForUser and it takes one argument. User is the type of the argument

Reply:
I'm still a bit confused on when we want to use the singleton pattern vs the static pattern.
@jonwis FYI

Reply:
RECOMENDED: Leave as is. This new type does not have the ...ForUser variant anyway.